modules/edu.gemini.seqexec.server/build.sbt

```diff
@@ -0,0 +1,20 @@
+import Settings._
+import Settings.Libraries._
+import Settings.LibrariesJVM._
+
+name := "edu.gemini.seqexec.server"
+
+libraryDependencies ++= Seq(
+  ScalaZCore,
+  ScalaZConcurrent,
+  "io.argonaut"        %% "argonaut"                  % "6.1",
+  "commons-httpclient" % "commons-httpclient"         % "2.0",
+  // OCS bundles
+  SpModelCore,
+  POT,
+  EpicsACM
+)
```

Comment:
I like these dependency variables like `ScalaZCore`.  Is there a reason to treat `argonaut` and `httpclient` differently?